include/win32/windows.h

```diff
@@ -1147,8 +1175,12 @@ void WINAPI OutputDebugStringA(
 void WINAPI OutputDebugStringW(
         LPCWSTR lpOutputString);
 BOOL WINAPI GetThreadContext(
-        HANDLE      hThread,
-        LPCONTEXT   lpContext);
+        HANDLE      hThread
+#if defined(_WIN32) || defined(_WIN64)
+// FIXME: Find LPCONTEXT Alternative
+    , LPCONTEXT   lpContext
+#endif
+);
```

Comment:
This will create an invalid API call. An alternative would be to define `LPCONTEXT` as `void *`.